src/components/transport_manager/test/tcp_transport_adapter_test.cc

```diff
@@ -259,34 +254,38 @@ TEST_F(TcpAdapterTest, StoreData_ConnectionNotExist_DataNotStored) {
   transport_adapter.CallStore();
 
   // Check that value is not saved
-  Json::Value& tcp_dict =
-      last_state_.dictionary["TransportManager"]["TcpAdapter"]["devices"];
-  ASSERT_TRUE(tcp_dict.isNull());
+  utils::json::JsonValue dict = last_state_.dictionary();
+  utils::json::JsonValue tcp_dict =
+      dict["TransportManager"]["TcpAdapter"]["devices"];
+  ASSERT_TRUE(tcp_dict.IsNull());
 }
 
 TEST_F(TcpAdapterTest, RestoreData_DataNotStored) {
-  Json::Value& tcp_adapter_dictionary =
-      last_state_.dictionary["TransportManager"]["TcpAdapter"];
-  tcp_adapter_dictionary = Json::Value();
+  utils::json::JsonValue dict = last_state_.dictionary();
+  utils::json::JsonValue tcp_adapter_dictionary =
+      dict["TransportManager"]["TcpAdapter"];
+  tcp_adapter_dictionary = utils::json::JsonValue();
   MockTCPTransportAdapter transport_adapter(
       port, last_state_, transport_manager_settings);
   EXPECT_CALL(transport_adapter, Connect(_, _)).Times(0);
   EXPECT_TRUE(transport_adapter.CallRestore());
 }
 
-TEST_F(TcpAdapterTest, StoreDataWithOneDevice_RestoreData) {
+TEST_F(TcpAdapterTest, DISABLED_StoreDataWithOneDevice_RestoreData) {
   MockTCPTransportAdapter transport_adapter(
       port, last_state_, transport_manager_settings);
   std::string uniq_id = "unique_device_name";
-  utils::SharedPtr<MockTCPDevice> mockdev = new MockTCPDevice(port, uniq_id);
+  utils::SharedPtr<MockTCPDevice> mockdev =
+      MakeShared<MockTCPDevice>(host_address, uniq_id);
   transport_adapter.AddDevice(mockdev);
 
   std::vector<std::string> devList = transport_adapter.GetDeviceList();
   ASSERT_EQ(1u, devList.size());
   EXPECT_EQ(uniq_id, devList[0]);
 
   const int app_handle = 1;
-  std::vector<int> intList = {app_handle};
+  std::vector<int> intList;
```

Comment:
@OHerasym May be use vector ctor with parameters? I mean `std::vector<int> intList(1, app_handle) ` 
What do you think?